README.md

```diff
@@ -120,6 +123,113 @@ and it should work.
 
 These rules are not supported by Google's Go team.
 
+<a name="go_repositories"></a>
+## go\_repositories
+
+```bzl
+go_repositories()
+```
+
+Instantiates external dependencies to Go toolchain in a WORKSPACE.
+All the other workspace rules and build rules assume that this rule is
+placed in the WORKSPACE.
+
+
+<a name="go_repository"></a>
+## go\_repository
+
+```bzl
+go_repository(name, importpath, revision)
```

Comment:
As an interface, it sounds good.
But as implementation `fetch_repo` cannot distinguish tags from commits because it relies on [`Cmd.CreateAtRev`](https://godoc.org/golang.org/x/tools/go/vcs#Cmd.CreateAtRev).

Is this acceptable for you?